src/components/breadcrumbs/index.md.njk

```diff
@@ -10,8 +10,12 @@ copy: done
 layout: layout-pane.njk
 ---
 
+{% from "_example.njk" import example %}
+
 The Breadcrumbs component helps users to understand where they are within a website’s structure and move between levels.
 
+{{ example({group: 'components', item: 'breadcrumbs', example: 'default'}) }}
```

Comment:
Presumably the components / item are needed because the helper doesn't know what file we're 'in'? i.e. it can't just import relatively?

Reply:
If so… could we set the path as a global when we're parsing each file, and modify the example macro to use that?

Reply:
I thought about that, but that would limit us to use examples only from this directory. If we want to have a breadcrumb example on the navigation pattern how do we achieve that?

Reply:
Hmm, I'd assume we'd create an example for that pattern specifically. I think that would probably be the safer way of creating examples, otherwise it would be easy to e.g. update a breadcrumb example without realising it's used elsewhere.